[bullet3] build failure on <any>-android with ndk 27

### Operating system

Linux

### Compiler

Clang

### Steps to reproduce the behavior

```Shell
# Download android ndk r27 from https://developer.android.com/ndk/downloads
export ANDROID_NDK_HOME=$HOME/Downloads/android-ndk-r27/
run `vcpkg install miniz:arm64-android`
```


### Failure logs

```
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is Clang 18.0.1
-- The CXX compiler identification is Clang 18.0.1
CMake Warning (dev) at /home/sandy/Downloads/android-ndk-r27/build/cmake/flags.cmake:46 (if):
  Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
  --help-policy CMP0057" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/Platform/Android-Clang.cmake:23 (include)
  /usr/share/cmake/Modules/Platform/Android-Clang-C.cmake:1 (include)
  /usr/share/cmake/Modules/CMakeCInformation.cmake:48 (include)
  CMakeLists.txt:7 (PROJECT)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at /home/sandy/Downloads/android-ndk-r27/build/cmake/flags.cmake:46 (if):
  if given arguments:

    "hwaddress" "IN_LIST" "ANDROID_SANITIZE"

  Unknown arguments specified
Call Stack (most recent call first):
  /usr/share/cmake/Modules/Platform/Android-Clang.cmake:23 (include)
  /usr/share/cmake/Modules/Platform/Android-Clang-C.cmake:1 (include)
  /usr/share/cmake/Modules/CMakeCInformation.cmake:48 (include)
  CMakeLists.txt:7 (PROJECT)
```

### Additional context

https://github.com/android/ndk/issues/2032

--- a/ports/bullet3/portfile.cmake
+++ b/ports/bullet3/portfile.cmake
@@ -32,6 +32,7 @@ vcpkg_cmake_configure(
         -DBUILD_GIMPACTUTILS_EXTRA=OFF        
         -DBUILD_UNIT_TESTS=OFF        
         -DINSTALL_LIBS=ON
+        -DCMAKE_POLICY_DEFAULT_CMP0057=NEW
         ${FEATURE_OPTIONS}
 )
 

--- a/ports/bullet3/vcpkg.json
+++ b/ports/bullet3/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "bullet3",
   "version": "3.25",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Bullet Physics is a professional collision detection, rigid body, and soft body dynamics library",
   "homepage": "https://github.com/bulletphysics/bullet3",
   "license": "Zlib",

--- a/versions/b-/bullet3.json
+++ b/versions/b-/bullet3.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8ee7660a10459f4d3d11054ab88177126d2131ca",
+      "version": "3.25",
+      "port-version": 2
+    },
+    {
       "git-tree": "9ecadb6b6e0651347e89555207f28c4ee5b1b51c",
       "version": "3.25",
       "port-version": 1

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1410,7 +1410,7 @@
     },
     "bullet3": {
       "baseline": "3.25",
-      "port-version": 1
+      "port-version": 2
     },
     "bustache": {
       "baseline": "1.1.0",

